src/crimson/common/config_proxy.h

```diff
@@ -28,10 +32,10 @@ class ConfigProxy : public seastar::peering_sharded_service<ConfigProxy>
   ObserverMgr<ConfigObserver> obs_mgr;
 
   const md_config_t& get_config() const {
-    return remote_config ? *remote_config : * local_config;
```

Comment:
Probably ought to be in its own commit since it's just a cosmetic change.

Reply:
A general question: I'd prefer doing those minor changes as part of a larger commit, as otherwise
the effort is too large.

Reply:
FWIW, maybe you could use `git add -p` to make this process simpler?

Reply:
I find that using git gui makes separating out little patches like that to be very quick and it makes future bug investigations a lot easier, but whatever you're comfortable with I guess.